src/App.vue

```diff
@@ -319,4 +319,17 @@ body {
   width: 100%;
   bottom: 0;
 }
+
+.button-return {
```

Comment:
aqui seria bom você criar o outline no `is-info` e daí usar essa classe

Reply:
Eu alterei o botão com o outlined, acredito que ficou como você falou, retirei a classe return-button.